jablib/src/test/java/org/jabref/logic/util/ExternalLinkCreatorTest.java

```diff
@@ -29,25 +50,34 @@ private boolean urlIsValid(String url) {
 
     @Test
     void getShortScienceSearchURLEncodesSpecialCharacters() {
+        ImporterPreferences stubPreferences = new StubImporterPreferences();
```

Comment:
Typically, this is done with `mock` and `when`. See other classes. Can you try this?